docs/view-managers.md

```diff
@@ -28,6 +28,11 @@ Currently, React Native for Windows uses a global `PerspectiveTransform` to prov
 
 __Important__: The `IsPerspectiveEnabled` property is experimental and support for it may be removed in the future.
 
+### Note about wrapping Windows controls
+As described above, some native UI components such as `ScrollView` and `TextInput` have been built by wrapping native Windows controls from the UWP XAML controls library. As a result, their visual appearance, animations, etc. will always match their native Windows counterpart. UI components can also be built using Javascript. In this case, the component's source code does not wrap a native control, but rather combines already contructed React Native Windows components to build itself. Components built in this way contain no C++/C# code.

```

Comment:
```suggestion
As described above, some native UI components such as `ScrollView` and `TextInput` have been built by wrapping native Windows controls from the UWP XAML controls library. As a result, their visual appearance, animations, user experience, accessibility, etc. will always match their native Windows counterpart. 

UI components can also be built using Javascript. In this case, the component's source code does not wrap a native control, but rather combines already constructed React Native Windows components to build itself. Components built in this way contain no native code.

```

Reply:
It'll probably be a good idea to have some blurb about the various pros/cons of the two approaches. For example, wrapping will provide full-fidelity native code, but trade-off cross-platform user experience. JS controls can be service-delivered and styled using CSS concepts, but lack full native fidelity and require you to build your own accessibility solutions.

Reply:
```suggestion
As described above, some native UI components such as `ScrollView` and `TextInput` have been built by wrapping native Windows controls from the UWP XAML controls library. As a result, their visual appearance, animations, user experience, accessibility, etc. will always match their native Windows counterpart. 

UI components can also be built using JavaScript. In this case, the component's source code does not wrap a native control, but rather combines already constructed React Native Windows components to build itself. Components built in this way contain no native code. However, components built this way may not be as fully-featured or match the look and feel of the operating system the app is running on.
```